lib/src/request.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:typed_data';
 import 'dart:convert';
 
+// ignore: import_of_legacy_library_into_null_safe
 import 'package:http/http.dart' as http;
 import 'package:meta/meta.dart';
```

Comment:
Do we still need `meta` as a dependancy now that `required` is built into the lang?

Reply:
It's needed for the `@visibleForTesting` annotation.